.github/workflows/on_pr_push.yml

```diff
@@ -90,6 +90,10 @@ jobs:
         PGSSLKEY: ""
         PGSSLCERT: ""
         PGSSLROOTCERT: ""
+
+        # HANA tests
+        RUN_HANA_TESTS: yes
+        HANA_CONNECTION_STRING: "hdb://TEGOLACI:iZgd6$nOdhf@917df316-4e01-4a10-be54-eac1b6ab15fb.hana.prod-us10.hanacloud.ondemand.com:443?TLSInsecureSkipVerify&TLSServerName=host&timeout=1000&max_connections=10"
```

Comment:
It looks like there is a username and password in the URL?  If there is, we should move it to GitHub secret; so that it's not leaked in a public repo like this?

Reply:
The credentials were exposed on purpose. We use the same HANA instance in other projects like GDAL and QGIS, where data source providers for other databases (Postgresql, Oracle, etc.) are installed in a CI environment. This is rather tricky and resource/time-consuming in the case of HANA. So we decided to install a HANA instance that is used in different OS projects in their CIs. If we move the credentials to GitHub secret, then CI tests for HANA will fail in forked repositories.